File: 1704-special-positions-in-a-binary-matrix/special-positions-in-a-binary-matrix.java
class Solution {
    public int numSpecial(int[][] mat) {
        int count=0;
        for(int []row:mat){
            if(rowsum(row)==1){
                count+=(getcolumnsum(mat,getindex(row))==1)?1:0;
            }
        }
        return count;
    }
    public static int rowsum(int row[]){
        return Arrays.stream(row).sum();
    }
    public static int getindex(int row[]){
        for(int i=0;i<row.length;i++){
            if(row[i]==1) return i;
        }
        return -1;
    }
    public static int getcolumnsum(int mat[][],int index){
        int sum=0;
        for(int row[]:mat){
            sum+=row[index];
        }
        return sum;
    }
}

File: 1704-special-positions-in-a-binary-matrix/special-positions-in-a-binary-matrix.py
class Solution(object):
    def numSpecial(self, mat):
        """
        :type mat: List[List[int]]
        :rtype: int
        """
        count=0
        for i in mat:
            if sum(i)==1:
                count+=1 if self.getcolumnsum(mat,i.index(1))==1 else 0
        return count
    def getcolumnsum(self,mat,index):
        sm=0
        for i in mat:
            sm+=i[index]
        return sm